### PR TITLE
fix(design-system): add missing export for Avatar component

### PR DESCRIPTION
**Pull request by stdavis**

### CODE REVIEW

File: packages/utah-design-system/src/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+export * from './components/Avatar.jsx';
```
Comment (steveoh) [inline packages/utah-design-system/src/index.js:1]: This is a tsx file right?

Comment (stdavis) [reply packages/utah-design-system/src/index.js]: Good catch.

**Review by steveoh — APPROVED** [review]
I wonder if this will fix the type issue?

**Comment by stdavis** [comment]
> I wonder if this will fix the type issue?

It did not.

**Comment by steveoh** [comment]
Yeah it still has that weird package folder structure.

**Pull request merged**